### PR TITLE
AAP-743: Revised subtitle and fixed broken jira link

### PR DESCRIPTION
**Pull request by lmalivert**
Jira: https://issues.redhat.com/browse/AAP-743

- Fixed the broken link in the docinfo.xml file. Should be `<link xlink:href="https://issues.redhat.com"></link>.`
- Revised subtitle to describe the purpose of the guide.

### CODE REVIEW

File: downstream/titles/security-guide/docinfo.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <title>Red Hat Ansible Security Automation Guide</title>
 <productname>Red Hat Ansible Automation Platform</productname>
 <productnumber>2.0</productnumber>
```
Comment (TarikFD) [inline downstream/titles/security-guide/docinfo.xml:3]: Is this title meant for 2.0?

**Review by TarikFD — CHANGES_REQUESTED** [review]
The docinfo.xml builds correctly for me, and the language change makes sense. Just had one question about version number, but otherwise LGTM.

File: downstream/titles/security-guide/docinfo.xml (hunk 1 above)
Comment (lmalivert) [reply downstream/titles/security-guide/docinfo.xml:3]: Thanks for the review, @TarikFD. This fix is to be backported into 2.1.
Good catch!

**Review by ariordan-redhat — APPROVED** [review]
Looks good to me

**Comment by lmalivert** [comment]
Changed the version number...can you please review @TarikFD or @ariordan-redhat. 
Thanks in advance!

**Comment by lmalivert** [comment]
@ariordan-redhat I'm gonna go ahead and delete the branch. Fine w/you?

**Comment by ariordan-redhat** [comment]
Backported to 2.1 ([PR#252](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/pull/252))

**Pull request merged**